flink-libraries/flink-sql-client/pom.xml

```diff
@@ -45,22 +45,19 @@ under the License.
 
 		<dependency>
 			<groupId>org.apache.flink</groupId>
-			<!-- use a dedicated Scala version to not depend on it -->
```

Comment:
I am puzzled by these dependencies. Why does the SQL client create an artifact that is not Scala versioned when it has "compile" dependencies on Scala-versioned artifacts? This seems like a problem in the current setup that is only not surfacing because there is only one Scala version. As soon as we support a second version, this will cause real problems with non-deterministic dependencies of released artifacts.